### ISSUE TITLE
Unittests cannot be run more than once without cleaning .pyc files

### ISSUE THREAD

**Issue by BallisticBuddha** [acoomans/flask-autodoc#22]
This is mainly an issue for local development by running the unittests locally.
Found with Python 2.7.6

To reproduce:

* from the root of the project directory, run `python -m unittest discover`
* from the same place, without altering the unittest file `test_autodoc.py` file, run `python -m unittest discover` again

This results in the following error:

```.F......F
======================================================================
FAIL: testGet (tests.test_autodoc.TestAutodoc)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/alex/git/flask-autodoc/tests/test_autodoc.py", line 33, in testGet
    self.assertIn(__file__, d['location']['filename'])
AssertionError: '/home/alex/git/flask-autodoc/tests/test_autodoc.pyc' not found in '/home/alex/git/flask-autodoc/tests/test_autodoc.py'

======================================================================
FAIL: testPost (tests.test_autodoc.TestAutodoc)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/alex/git/flask-autodoc/tests/test_autodoc.py", line 53, in testPost
    self.assertIn(__file__, d['location']['filename'])
AssertionError: '/home/alex/git/flask-autodoc/tests/test_autodoc.pyc' not found in '/home/alex/git/flask-autodoc/tests/test_autodoc.py'
```

This is due to the tests that verify the location property being run from the compiled .pyc bytecode, therefore making the source of the actual file running the test the .pyc file, which appears as the `__file__` attribute in this scenario. 

I can see three possible solutions to this:

* Ignore the last character from the value being asserted if it is a 'c' character.
* Swap the values being asserted, making the line become `self.assertIn(d['location']['filename'], __file__)`. This isn't as clear when read, but basically because the expected  value '/path/to/project/test/test_autodoc.py' is contained in the string '/path/to/project/test/test_autodoc.pyc', it should pass the assertion.
* Run the unittests from a Makefile or similar that does a `clean` of the environment by removing excess artifacts such as .pyc files.

**Comment by lukeyeager** [comment]
Thanks for the detailed bug report. I actually ran into this myself while testing #19. Here's how I fixed it in that pull request - what do you think?

```python
    @staticmethod
    def thisFile():
        """Returns the basename of __file__ without a trailing 'c'"""
        filename = os.path.basename(__file__)
        if filename.endswith('.pyc'):
            filename = filename[:-1]
        return filename

#...

        self.assertIn(self.thisFile(), d['location']['filename'])
```

**Comment by BallisticBuddha** [comment]
Yep, that looks good, and is what I was talking about with my first suggestion by ignoring the last character in this instance.

**Comment by lukeyeager** [comment]
Closed in #19.
